security-oauth2/build.gradle

```diff
@@ -2,6 +2,11 @@ plugins {
     id "io.micronaut.build.internal.module"
 }
 
+repositories {
+    mavenCentral()
```

Comment:
`mavenCentral` is not necessary. I think that It is added as a repository by `io.micronaut.build.internal.module`

Reply:
I've done this change.